beastify/popup/choose_beast.js

```diff
@@ -65,13 +65,13 @@ function listenForClicks() {
      * Get the active tab,
      * then call "beastify()" or "reset()" as appropriate.
      */
-    if (e.target.type === "reset") {
+    if (e.target.type === "button") {
```

Comment:
Instead of being indirect here, let's just prepend:

```lang=javascript
if (e.target.tagName !== "BUTTON" || !e.target.closest("#popup-content")) {
  // Ignore when click is not on a button within <div id="popup-content">.
  return;
}
```